### PR TITLE
Allow label selection on attribute extraction tasks

### PR DESCRIPTION
**Pull request by rajasbansal**
Allows using label selection if specified for attribute extraction tasks

### CODE REVIEW

File: src/autolabel/few_shot/vector_store.py [hunk 1]
```diff
@@ -17,13 +17,6 @@
 
 logger = logging.getLogger(__name__)
 
-try:
-    import torch
```
Comment (nihit) [inline src/autolabel/few_shot/vector_store.py:21]: should we remove this requirement from pyproject.toml ?

**Review by nihit — APPROVED** [review]
